cfep-03.md

```diff
@@ -0,0 +1,51 @@
+
+<table>
+<tr><td> Title </td><td> Providing manually uploaded builds on the conda-forge anaconda channel</td>
+<tr><td> Status </td><td> Proposed </td></tr>
+<tr><td> Author(s) </td><td> Hadrien Mary &lt;hadrien.mary@gmail.com&gt;</td></tr>
+<tr><td> Created </td><td> Sept 27, 2016</td></tr>
+<tr><td> Updated </td><td> Sept 27, 2016</td></tr>
+<tr><td> Discussion </td><td> NA </td></tr>
+<tr><td> Implementation </td><td> NA </td></tr>
+</table>
+
+## Abstract
+
+Some packages are very hard to build on the currently used CI (circleci, Travis and AppVeyor). It's most of the time because of time limitations in the CI website and also because of the complexity of the build.
+
+An example is the Qt5 build that seems to be very hard to build (it tooks almost a year for Continuum folks to compile it for all platforms). See the Qt5 recipe build discussion ticket here https://github.com/conda-forge/qt-feedstock/issues/5.
+
+That being said it's excepted that a small amount of packages will be very hard to automatically build for various reasons. 
+
+Obviously we all want to keep this amount of packages small and have a maximun of automatically build packages on conda-forge.
+
+But a system/protocol to manually upload builds on conda-froge anaconda channel is necessary to circumvent some situations.
+
+## Specification
+
+A protocol to manually upload builds could look something like that :
+
+- Manually build the different packages for all targeted platforms.
```

Comment:
Could we outline a bit more the steps that go into this? For instance, on Linux we can use our Docker container to build packages. On Windows, maybe we could use this ( https://github.com/ContinuumIO/vagrant-images/pull/1 ). On OS X, I'm not sure what we can do ATM, but it would be nice if we could give that some thought too.

Reply:
In the new CEP-03 this points to the docs that are more dynamic and avoid duplication.